### PR TITLE
Toggle switch error messages

### PR DESCRIPTION
**Pull request by camertron**
### Description

@neall and I have been pairing on getting error messages to display properly when a toggle switch request fails. This is the result.

image

### Integration

> Does this change require any updates to code in production?

No

### Merge checklist

- [x] Added/updated tests
~- [ ] Added/updated documentation~
- [x] Added/updated previews

### CODE REVIEW

**Comment by camertron** [comment]
Visual differences are expected.

File: app/components/primer/alpha/toggle_switch.pcss [hunk 1]
```diff
@@ -208,6 +208,11 @@
   text-align: right;
 }
 
+.ToggleSwitch-statusIcon {
+  width: 16px;
+  height: 16px;
```
Comment (jonrohan) [inline app/components/primer/alpha/toggle_switch.pcss:213]:
```suggestion
  width: var(--base-size-16, 16px);
  height: var(--base-size-16, 16px);
```

**Review by jonrohan — COMMENTED** [review]
Just one css request

File: app/components/primer/alpha/toggle_switch.pcss (hunk 1 above)
Comment (camertron) [reply app/components/primer/alpha/toggle_switch.pcss:213]: This div wraps a couple of octicons, so I think we explicitly want 16px of width and height?

**Comment by tobiasahlin** [comment]
Should the octagon be nudged 1-2 pixels upwards, perhaps? 1px up:
test

File: app/components/primer/alpha/toggle_switch.pcss [hunk 2]
```diff
@@ -208,6 +208,11 @@
   text-align: right;
 }
 
+.ToggleSwitch-statusIcon {
+  width: 16px;
```
Comment (langermank) [inline app/components/primer/alpha/toggle_switch.pcss:212]:
```suggestion
  width: var(--base-size-16, 16px);
```

**Review by jonrohan — APPROVED** [review]

**Pull request merged**